app/controllers/admin/projects_controller.rb

```diff
@@ -19,9 +19,69 @@ def show
     }
   end
 
+  def new
+    @project = Project.new
+    render json: @project
+  end  
+
+   
+  def create
+    @project = Project.new(project_params)
+    if @project.save
+      render json: {
+      data: ActiveModelSerializers::SerializableResource.new(@project, serializer: ProjectSerializer),
+      message: ['Project created successfully'],
+      status: 200,
+      type: 'Success'
```

Comment:
use rails status codes  http://www.railsstatuscodes.com/
Example : render json: { success :true}, status: :ok

Reply:
@saikrishnapujari97 I think, here the status should be 201. status: :created